ffmpeg/signcomp_test.go

```diff
@@ -0,0 +1,71 @@
+package ffmpeg
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestCompareSignData(t *testing.T) {
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	var device = []string{"sw", "nv"}
+	fnames := make([]string, 0)
+	for _, dev := range device {
+		for i := 1; i <= 2; i++ {
+			vname := fmt.Sprintf("sign_%s%d.bin", dev, i)
+			fname := path.Join(wd, "..", "data", vname)
+			fnames = append(fnames, fname)
+		}
+	}
+
+	fnames = append(fnames, path.Join(wd, "..", "data", "nodata.bin"))
+
+	res, err := CompareSignatureByPath(fnames[0], fnames[2])
```

Comment:
nit: IMO it would be simpler to understand the code if we read the signature binary files into variables that represent what they are, so that our comparisons look like: `CompareSignatureByPath(sw_1_sig, nv_1_sig)` etc.